community/releases/release-manager-runbook/index.md

```diff
@@ -248,11 +276,16 @@ Example: VERSION="1.17.2" ./publish.sh
 
 Repeat weeklyish for each supported version.
 
-1. Check for any PRs waiting to be [cherry-picked](https://github.com/search?q=org%3Aspinnaker+state%3Aopen+author%3Aapp%2Fmergify+is%3Apr&type=Repositories&ref=advsearch&l=&l=).
+1. Check for any PRs waiting to be cherry-picked.
 Ensure patches meet the
 [release branch patch criteria](/community/contributing/releasing#release-branch-patch-criteria)
 before merging.
 
+    You can use this search to see what's waiting to be cherry-picked
+    ```
+    https://github.com/search?type=Issues&q=org:spinnaker+state:open+is:pr+base:${RELEASE_BRANCH}
```

Comment:
If we're going to include the cherry-pick search link as this fill-in-the-blank URL, let's clarify that release managers will need to construct three separate queries, one for each supported $RELEASE_BRANCH.